apps/ecs/behaviours/send-email-notification.js

```diff
@@ -0,0 +1,141 @@
+const config = require('../../../config');
+
+const NotifyClient = require('notifications-node-client').NotifyClient;
+const notifyKey = config.govukNotify.notifyApiKey;
+const notifyClient = new NotifyClient(notifyKey);
+const moment = require('moment');
+const PRETTY_DATE_FORMAT = 'DD MMMM YYYY';
```

Comment:
I would suggest moving `PRETTY_DATE_FORMAT` to config for consistency and reusability